scripts/system/create/edit.js

```diff
@@ -2905,7 +2905,19 @@ function getExistingZoneList() {
         };
         listExistingZones.push(thisZone);
     }
+    listExistingZones.sort(GetSortOrder("name"));
```

Comment:
```suggestion
    listExistingZones.sort(zoneSortOrder);
```
Two things here:
- Sort function name should start with a lower case letter.
- Not passing the sort property into the function may speed up the sort?

(Set `var zoneSortProperty = "name";` outside function and update function code to use this variable.)